### PR TITLE
Fix a little typo in Announcing Hanami v2.0.0.alpha2

### PR DESCRIPTION
**Pull request by georgiybykov**
I think that the missed word is "application", however it will make a tautology. Thus, I propose to correct the typo and make a little change for the sentence structure.

### CODE REVIEW

File: source/blog/2021-05-04-announcing-hanami-200alpha2.html.markdown [hunk 1]
```diff
@@ -14,7 +14,7 @@ It’s been a little while since [the last alpha release](https://hanamirb.org/b
 
 - A **completely rewritten application core**, offering advanced application-level state management and code loading capabilities
 - An **always-there auto-injection mixin**, making it easy to model your behavior as functional, composable objects
-- Built-in **application settings**, providing first-class support for your
+- Built-in **application settings** that provide first-class support
```
Comment (adam12) [inline source/blog/2021-05-04-announcing-hanami-200alpha2.html.markdown:17]:
```suggestion
- Built-in **application settings** with first-class support
```

**Review by adam12 — COMMENTED** [review]
WDYT?

**Comment by georgiybykov** [comment]
> WDYT?

Nice catch, thanks!

**Comment by adam12** [comment]
Thanks for the patch!

**Pull request merged**